src/index.html

```diff
@@ -10,7 +10,46 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page__body">
+    <form
+      data-qa="big"
+      class="search-bar search-bar--big"
```

Comment:
```suggestion
      class="search-bar"
```
pay attention on this point from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md). same for other elements
![Screenshot from 2023-09-08 13-16-42](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/c55a5776-5e46-4262-82f4-010213a85992)